tendrl/commons/flows/import_cluster/gluster_help.py

```diff
@@ -147,31 +147,40 @@ def import_gluster(parameters):
         )
         err, out, rc = command.run()
         if err:
+            err_msg = "Could not enable gluster-integration " \
```

Comment:
"service tendrl-gluster-integration" instead of  just ?"gluster-integration"

Reply:
Done